curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-25-5-clock.md

```diff
@@ -18,7 +18,7 @@ You can use any mix of HTML, JavaScript, CSS, Bootstrap, SASS, React, Redux, and
 
 **User Story #2:** I can see an element with `id="session-label"` that contains a string (e.g. "Session Length").
 
-**User Story #3:** I can see two clickable elements with corresponding IDs: `id="break-decrement"` and `id="session-decrement"`.
+**User Story #3:** I can see two clickable elements with corresponding IDs: `id="break-decrement"` and `id="break-decrement"`.
```

Comment:
This doesn't seem to be a typo. The story expects a "break-decrement" and "session-decrement" button - two buttons that reduce the break time and session time.

This is related to user story 4, which expects "break-increment" and "session-increment" for increasing the times.